github-notifications/index.js

```diff
@@ -87,18 +46,49 @@ async function watch({ snapshot, auth: { token }, libs: { axios } }) {
       };
     }
 
-    if (status === 401) {
-      error.key = ERRORS.auth;
-    } else {
-      error.key = ERRORS.other;
-    }
+    throw error.response;
+  }
 
-    return {
-      snapshot,
-      notifications: [],
-      error,
-    };
+  const { data, headers } = response;
+  const notifications = [];
+
+  if (data.length > 0) {
```

Comment:
What if `data === undefined`?

Reply:
it would mean that the GitHub API had a breaking change